### PR TITLE
Improve Grafana dashboards

### PR DESCRIPTION
**Pull request by oxddr**
* use new metrics
* add concise legends

### CODE REVIEW

File: clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py [hunk 1]
```diff
@@ -148,43 +173,57 @@
...
+        legend="{{instance}}: {{version}}.{{group}}.{{kind}}",
     ),
     d.simple_graph(
         "Watch events rate",
-        "sum(irate(apiserver_watch_events_total[1m])) by (version, kind, instance)",
+        "sum(irate(apiserver_watch_events_total[1m])) by (instance, group, version, kind)",
+        legend="{{instance}}: {{version}}.{{group}}.{{kind}}",
     ),
     d.simple_graph(
         "(Experimental) Watch events traffic",
```
Comment (mborsz) [inline clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: You can remove (Experimental). It's merged already and personally I find this very useful metric.

File: clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py [hunk 2]
```diff
@@ -18,28 +18,49 @@
...
-        'histogram_quantile(0.99, sum(rate(apiserver_request_duration_seconds_bucket{verb!~"LIST|WATCH|CONNECT|DELETECOLLECTION"}[1d])) by (le, resource, verb, scope, subresource))',
-        legend="{{verb}} {{scope}}/{{resource}}",
-        yAxes=g.single_y_axis(format=g.SECONDS_FORMAT),
-    ),
-    d.simple_graph(
-        "API call latency aggregated (prometheus, 1s threshold)",
-        'quantile_over_time(0.99, apiserver:apiserver_request_latency:histogram_quantile{verb!~"LIST|WATCH|CONNECT|DELETECOLLECTION"}[1d])',
-        legend="{{verb}} {{scope}}/{{resource}}",
-        yAxes=g.single_y_axis(format=g.SECONDS_FORMAT),
+        'sum(rate(apiserver_request_total{client="clusterloader/v0.0.0 (linux/amd64) kubernetes/$Format", verb!="CONNECT"}[1m])) by (verb, resource)',
```
Comment (mborsz) [inline clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: sum(irate(apiserver_request_total))?

I was debugging for an hour why my test is overloaded at the very beginning and changing to irate allowed me to discover that I'm creating ~1K QPS of some objects creation.

using rate() was hiding this issue

**Comment by mborsz** [comment]
I added two comments. Other than that LGTM

**Comment by mm4tt** [comment]
Could you set-up a grafana for this change? You could use any recent 5K or 100 snapshot. That would make this change review-able :)

I'm asking because it happened already a few times that we accidentally broke some grafana plots, these changes are really hard to review if I cannot see the results in grafana

**Comment by oxddr** [comment]
> Could you set-up a grafana for this change? You could use any recent 5K or 100 snapshot. That would make this change review-able :)
> 
> I'm asking because it happened already a few times that we accidentally broke some grafana plots, these changes are really hard to review if I cannot see the results in grafana

I keep using this consoles locally, I'll DM you a link to the running instance.

**Comment by mm4tt** [comment]
/lgtm

/hold 
Adding hold if Maciek has some more comments, feel free to void it when you're ready

**Pull request merged**